### ISSUE TITLE
Enable cancellable speech by default and move the setting to speech settings

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#15457]
### Is your feature request related to a problem? Please describe.
There are issues such as #11058 or #9442 which are fixed when enabling cancellable speech in NVDA advanced settings. Actually in PR #11266 cnacellable speech was enabled by default but only for a short period of time.
Also the setting is not flaged as experimental anymore, so it should be moved from advanced settings to speech settings, if the setting is needed at all.

### Describe the solution you'd like
The cancellable speech should be the default in NVDA stable versions and actually there is no setting needed for this whether yes or no. This fixes some bugs so it should be the default behavior without being able to change it.

### Describe alternatives you've considered
Still let the users to decide if they want to enable cancellable speech or not, although I don't see a reason why people should set it to "no".

**Comment by lukaszgo1** [comment]
Cancellable speech is enabled by default since  NVDA 2021.1, so this part of the issue is not valid.

**Comment by Adriani90** [comment]
No for stable it seems is not, it says default (no) at least from what I see when I install NVDA.Von meinem iPhone gesendetAm 16.09.2023 um 22:35 schrieb Łukasz Golonka ***@***.***>:﻿
Cancellable speech is enabled by default since  NVDA 2021.1, so this part of the issue is not valid.

**Comment by lukaszgo1** [comment]
I have re-tested by creating a clear copy of 2023.2, with a clean config, and on my side cancellable speech is enabled by default. The option in settings is set to 'Default (Yes)'. Could you check again, making sure you're using a clean config?

**Comment by Adriani90** [comment]
Ok it seems it was a translation issue, I used NVDA in german and the string is standard (nein) which is aparently wrong. I will solve this part in the translation and edit this issue accordingly.

**Comment by seanbudd** [comment]
This can be removed from the GUI.

We believe this is too technical for speech settings, so it should either remain or be removed.

**Comment by XLTechie** [comment]
Should it be just removed from the GUI but left as a "hidden" option like 
several others, or removed completely as an option?

Personally I think that when things like this are removed from the GUI, they 
should be left as hidden (config-only) options until the next+1 breaking 
release, or until the associated code becomes burdensome to maintain.

**Comment by seanbudd** [comment]
We believe it should be fully removed. We currently make no efforts to maintain the configSpec to only change in a breaking release, and we want to actively drop support for the "no" case here.
